tty/usbacm/usbacm.c

```diff
@@ -196,26 +196,56 @@ static usbacm_dev_t *usbacm_get(int id)
 }
 
 
-static void _usbacm_put(usbacm_dev_t *dev)
+static void usbacm_remove(usbacm_dev_t *dev)
```

Comment:
I would name it `_free` to highlight that `dev` becomes invalid after calling this function